selenium_tests/management/commands/snapshot_dashboard_states.py

```diff
@@ -383,4 +427,4 @@ def handle(self, *args, **options):
         with override_settings(
             ELASTICSEARCH_INDEX='testindex',
         ):
-            sys.exit(pytest.main(args=["{}::DashboardStates".format(__file__), "-s"]))
+            sys.exit(pytest.main(args=["{}::test_dashboard_states".format(__file__), "-s", "--create-db"]))
```

Comment:
Do we still need the `--create-db` here? It looks like it's causing the management command to rerun migrations each time even if they can be skipped